contracts/registry/MagicDropTokenImplRegistry.sol

```diff
@@ -173,13 +194,32 @@ contract MagicDropTokenImplRegistry is UUPSUpgradeable, Ownable, IMagicDropToken
         }
     }
 
+    /// @dev Gets the mint fee for a given token standard
```

Comment:
personal/discussion nits: until we definitively know the gas savings we get for inlined assembly I think it makes sense to stick to the original Solidity ethos of "could be accessible to developers without specialized blockchain knowledge"

Reply:
I'm generally in agreement that inline assembly should be used sparsely or inherited from battle-tested libraries unless the gas savings are significant.

In this `getMintFee` case, I use it for uniformity with `getDeploymentFee`. Both getters should follow the same retrieval pattern.